packages/client/tests/functional/_utils/idForProvider.ts

```diff
@@ -0,0 +1,6 @@
+const sqlId = 'Int @id @default(autoincrement())'
```

Comment:
Maybe 

```suggestion
const sqlId = 'String @id @default(uuid())'
```

so that the type will be the same regardless of the provider